cli-output/src/cli_output.rs

```diff
@@ -2729,6 +2729,55 @@ impl fmt::Display for CliBlock {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CliEntries {
```

Comment:
nit: We do have an `output.rs` within `ledger-tool`; `ledger-tool` looks like the only caller so would this be better suited for that?

Reply:
🤦‍♀️ Oh right. Agreed, I will move.